example/android/app/src/main/kotlin/com/apptentive/apptentive_flutter_example/MainActivity.kt

```diff
@@ -1,11 +0,0 @@
-package com.apptentive.apptentive_flutter_example
-
-import io.flutter.embedding.android.FlutterActivity
-
-// Implement ApptentiveActivityInfo in order to show Apptenting Interactions
-// through the Apptentive Flutter Plugin
-class MainActivity : FlutterActivity() {
-    override fun configureFlutterEngine(@NonNull flutterEngine: FlutterEngine) {
-        GeneratedPluginRegistrant.registerWith(flutterEngine);
-    }
-}
```

Comment:
Curious why you replaced the Kotlin example with a Java one?

Reply:
example app had both. The Kotlin was not in use. I tried to keep the kotlin but wasn't able to make it work. However, FoxFacts uses kotlin Activity.